.github/workflows/mepbm.yml

```diff
@@ -0,0 +1,27 @@
+name: mepbm CI test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: install dependencies
+      run: |
+        sudo apt-get install doxygen graphviz libboost-all-dev numdiff -y
+        doxygen --version
+        git clone https://github.com/bangerth/SampleFlow.git
+    - name: cmake
+      run: cmake -DSAMPLEFLOW_DIR="$PWD/SampleFlow" .
+    -name: build
+      run: make
```

Comment:
It's not clear to me what would be wrong here. The line looks like all of the other `run:` lines that immediately list a command after the colon.